app/views/Dashboard/Overview/RegionalBreakdownCard/index.tsx

```diff
@@ -0,0 +1,165 @@
+                className={_cs(styles.globalStatCard)}
+                contentClassName={styles.responsiveContent}
+                heading="Total number of cases"
+                headingSize="extraSmall"
+                headerDescription="Loreum epsum epsum 2020"
+            >
+                <ResponsiveContainer className={styles.responsiveContainer}>
+                    <BarChart
+                        data={totalCasesBarChart}
+                        barSize={45}
+                    >
+                        <Tooltip
+                            isAnimationActive={false}
+                            allowEscapeViewBox={{
+                                x: false,
+                                y: false,
+                            }}
+                        />
+                        <XAxis
+                            dataKey="name"
+                            tickLine={false}
+                            axisLine={false}
+                        >
+                            <LabelList
+                                dataKey="name"
+                                position="bottom"
+                                fontSize="10"
+                            />
+                        </XAxis>
+                        <Bar
+                            dataKey="amt"
+                            isAnimationActive={false}
+                        >
+                            {totalCasesBarChart?.map((entry) => (
+                                <Cell
+                                    key={`Cell -${entry.id}`}
+                                    fill={COLORS[entry.id % COLORS.length]}
+                                />
+                            ))}
+                            <LabelList dataKey="range" position="top" />
+                        </Bar>
+                    </BarChart>
+                </ResponsiveContainer>
+            </ContainerCard>
+            <ContainerCard
+                className={_cs(styles.regionsPieChart)}
+                contentClassName={styles.responsiveContent}
+                heading="Regional Breakdown"
+                headingSize="extraSmall"
+                headerDescription="Loreum Ipsum epsum san-diego"
+            >
+                <div className={styles.pieChartCollection}>
+                    <List
+                        keySelector={pieChartInfoKeySelector}
+                        data={regionalBreakdownPieData}
+                        renderer={PieChartInfo}
+                        rendererParams={pieChartInfoRendererParams}
+                    />
+                </div>
+                <div className={styles.breakdownLabel}>
+                    <List
+                        keySelector={breakdownLabelKeySelector}
+                        data={regionalBreakdownPieData}
+                        renderer={RegionalBreakdownLabel}
+                        rendererParams={regionalBreakdownLabelParams}
+                    />
+                </div>
```

Comment:
use listview instead of adding list inside div

Reply:
Using ListView instead of List component disrupts the re-charts container